pyflakes/checker.py

```diff
@@ -701,14 +732,20 @@ def is_typing_overload(value, scope_stack):
     )
 
 
+@contextlib.contextmanager
+def enter_annotation(instance):
+    orig, instance._in_annotation = instance._in_annotation, True
+    try:
+        yield
+    finally:
+        instance._in_annotation = orig
```

Comment:
oops sorry should have mentioned this last review -- this probably makes more sense as a member function (that way `instance` can just be `self` -- less of a leaky abstraction that way too 👍)